pkg/operator/reports_test.go

```diff
@@ -92,3 +96,86 @@ func TestGetNextReportPeriod(t *testing.T) {
 		})
 	}
 }
+
+func TestReportRunningCondition(t *testing.T) {
+	const (
+		namespace      = "default"
+		testReportName = "test-report"
+		testQueryName  = "test-query"
+	)
+
+	schedule := &metering.ReportSchedule{
+		Period: metering.ReportPeriodCron,
+		Cron:   &metering.ReportScheduleCron{Expression: "5 4 * * *"},
+	}
+
+	reportStart := &time.Time{}
+	reportEndTmp := reportStart.AddDate(0, 1, 0)
+	reportEnd := &reportEndTmp
+
+	testTable := []struct {
+		name       string
+		report     *metering.Report
+		expectFlag bool
+	}{
+		{"new report returns false", testhelpers.NewReport(testReportName, namespace, testQueryName, reportStart, reportEnd, v1alpha1.ReportStatus{}, nil), false},
+		{"run-once report returns true", testhelpers.NewReport(testReportName, namespace, testQueryName, reportStart, reportEnd, v1alpha1.ReportStatus{
+			Conditions: []metering.ReportCondition{
+				{
```

Comment:
You could use `NewReportCondition` from https://github.com/operator-framework/operator-metering/blob/master/pkg/apis/metering/v1alpha1/util/report_util.go  instead of using a struct literal.